apk/apk.go

```diff
@@ -273,7 +270,7 @@ func createSignatureBuilder(digest []byte, info *nfpm.Info) func(*tar.Writer) er
 		if keyname == "" {
 			addr, err := mail.ParseAddress(info.Maintainer)
 			if err != nil {
-				return errors.Wrap(err, "key name not set and unable to parse maintainer mail address")
+				return fmt.Errorf("key name not set and unable to parse maintainer mail address: %w", err)
 			} else if addr.Address == "" {
 				return errors.New("key name not set and maintainer mail address empty")
```

Comment:
We could also use `fmt.Errorf` for non-wrapping errors for consistency. But that's just a thought, `errors.New` is also fine of course.